### ISSUE TITLE
NPM Package Not Precompiled

### ISSUE THREAD

**Issue by zolrath** [MandarinConLaBarba/react-tree-menu#19]
Hi,
When attempting to use this library in our project we were greeted with errors as Babel ignores node_modules during the compilation step.
From what I can tell published npm packages should be precompiled to standard .js format, is there any chance you can republish?

**Comment by MandarinConLaBarba** [comment]
Yeah, I can try to do this at some point...no time table though unfortunately as I'm pretty busy lately. PR welcome
